tests/test_transport_framing.py

```
@@ -0,0 +1,218 @@
+                        buf += chunk
+                    return buf
+
+                header = _read_exact(8)
+                if len(header) == 8:
+                    length = struct.unpack(">Q", header)[0]
+                    payload = _read_exact(length)
+                    if payload == b'{"type":"ping"}':
+                        resp = b'{"type":"pong"}'
+                        conn.sendall(struct.pack(">Q", len(resp)) + resp)
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)